Bottom: c650ce68bd1b95380244b74feaf0361cadb70f80
Top:    9ac79169608ef9af2e3bf70695faf3248cb9151e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 11:23:10 +0100

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 74426fc..47ba6b4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1806,7 +1806,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.6
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
